src/components/AuthorCardEditing.vue

```diff
@@ -229,13 +278,101 @@ export default defineComponent({
                 .filter(byError(errors.value))
                 .map(query => query.replace.message)
         })
+        const helpData = {
+            name: {
+                title: 'name',
+                url: [
+                    {
+                        text: 'Click here to see the documentation for name.',
+                        link: 'https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema-guide.md#definitionspersongiven-names'
+                    }
+                ],
+                description: 'The person\'s given names.',
+                examples: [
+                    'Jane',
+                    'John'
+                ]
+            },
+            lastName: {
+                title: 'lastNames',
```

Comment:
Our discussion on #646 led to this `title` being the name of the field.
This doesn't work here and it badly applies in other places like Identifiers.
We can merge this first and discuss how to proceed later, but we should create an issue from this comment.

Reply:
I updated the title to include all of the fields.